src/components/attribute-input/AttributeInput.tsx

```diff
@@ -58,63 +58,95 @@ export const AttributeInput = ({
     }
   }, []);
 
-  const [isOpenArray, setIsOpenArray] = useState<boolean[]>([false]);
+  const [isKeyOpenArray, setIsKeyOpenArray] = useState<boolean[]>([false]);
```

Comment:
I think Typescript can infer the type here from the assigned value.
```suggestion
  const [isKeyOpenArray, setIsKeyOpenArray] = useState([false]);
```